core/web/src/main/java/org/opennms/core/web/HttpClientWrapper.java

```diff
@@ -388,7 +415,7 @@ public CloseableHttpClient getClient() {
                 requestConfigBuilder.setConnectTimeout(m_connectionTimeout);
             }
             if (m_retries != null) {
-                httpClientBuilder.setRetryHandler(new DefaultHttpRequestRetryHandler(m_retries, false));
+                httpClientBuilder.setRetryHandler(new HttpRequestRetryOnExceptionHandler(m_retries, true));
```

Comment:
This client is used in many different contexts, I'm afraid that flipping requestSentRetryEnabled from false to true might have some unintended consequences.